semanticdb-javac/src/main/java/com/sourcegraph/semanticdb_javac/SemanticdbVisitor.java

```diff
@@ -74,6 +74,7 @@ public SemanticdbVisitor(
 
   private <T extends JCTree & JCDiagnostic.DiagnosticPosition> void emitSymbolOccurrence(
       Symbol sym, T posTree, Role role, CompilerRange kind) {
+    if (sym == null) return;
```

Comment:
Is this an issue you came across?

Reply:
Yeah, hit on three NPEs when compiling the JDK